src/common/images/index.ts

```diff
@@ -0,0 +1 @@
+export { ReactComponent as AlkemioLogo } from './alkemio-logo.svg';
```

Comment:
The thing is imports outside of the **src** are not allowed and those images belong to a common/shared folder. to be reused. I created this to use the image inside the error page, but since we didn't have a design and me facing css responsive issues decided to scrape it.